src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,49 @@
+.nav {
+  &__list {
+    display: flex;
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    margin-left: 20px;
+    height: 60px;
+    display: flex;
+
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+
+  &__link {
+    font-weight: 500;
+    text-transform: uppercase;
+    font-size: 12px;
+    line-height: 14px;
+    color: $main-color;
+    align-content: center;
+    text-decoration: none;
+    transition: color $transition-time;
+
+    &:hover {
+      color: $blue-color;
+    }
+
+    &.is-active {
+      color: $blue-color;
+      position: relative;
+
+      &::after {
+        content: '';
+        background-color: $blue-color;
```

Comment:
The variable `$blue-color` is used here but not defined in this file. Ensure it is defined in a global variables file and imported correctly.